packages/nextjs/src/config/webpack.ts

```diff
@@ -261,9 +261,9 @@ function checkWebpackPluginOverrides(
  */
 function shouldAddSentryToEntryPoint(entryPointName: string, isServer: boolean): boolean {
   return (
-    entryPointName === 'pages/_app' ||
+    entryPointName.includes('pages/_app') ||
```

Comment:
we're sure that this is safe? Wouldn't asserting this exists at the end of the string be most correct? 

I guess having `_app` as a folder will never probably happen.